src/App.js

```diff
@@ -1,23 +1,18 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import NavBar from './components/NavBar';
+import BookList from './components/BookList';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="container-fluid bg-dark bg-opacity-10 py-5">
+      <Router>
+        <NavBar />
+        <Routes>
+          <Route path="/" element={<BookList />} />
+          <Route path="/categories" />
```

Comment:
- [ ] Please create a component for categories and which should display a button for `check status` 
![image](https://user-images.githubusercontent.com/62705947/214955918-d9bbfa9b-2e70-4665-822e-e5e436b5c2b4.png)